test/blackbox-tests/test-cases/runtest-cmd.t

```diff
@@ -101,3 +101,7 @@ messages are informative enough.
   Error: "mytest1.t" was not found.
   Hint: did you mean mytest.t?
   [1]
+- Running a non-test file should give a suitable error message
+  $ dune test dune-project
+  Error: "dune-project" was not found.
```

Comment:
What about "`dune-project` is not a test file"? If the file is present, it might confuse the user, no?

Reply:
Yes, this is a confusing error message at the moment. Which is why it will be improved in a follow up PR. My goal here was not to improve the error message but stop the useless hint.

Reply:
This will become:
```
  $ dune test dune-project
  Error: "dune-project" does not match any known test.
  [1]
```
later.

Reply:
I see. Sorry for the misunderstanding on my side!